pkgs/development/tools/marksman/update.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=./. -i bash -p curl jq common-updater-scripts
+#shellcheck shell=bash
+
+set -eu -o pipefail
+shopt -s inherit_errexit
```

Comment:
```suggestion
```
We are not setting this in many update scripts so I am a bit reluctant to do it here.